pkg/assetfetcher/asset.go

```diff
@@ -352,3 +352,12 @@ func (a assetHistoryEvents) lastScannedTimestamp() (time.Time, error) {
 	// returned
 	return latestTime, nil
 }
+
+// AssetMissingField is called when an asset is skipped due to missing fields. It must either not have ID or not having Hostname or IP,
```

Comment:
question: _Must_ it be one or the other? Is it not possible that all three fields are missing?

Reply:
it is indeed possible! I can make maybe a third field and edit the comment